codersdk/roles.go

```diff
@@ -18,6 +18,13 @@ type SlimRole struct {
 	DisplayName string `json:"display_name"`
 }
 
+func (s SlimRole) String() string {
+	if s.DisplayName != "" {
+		return s.DisplayName
+	}
+	return s.Name
+}
```

Comment:
This is a nice way to make the table renderer handle this struct in a nice manner.